docs: update README (#235)
edgexfoundry/go-mod-messaging: README.md

The Message Bus connection information as well as which implementation to use is stored in the service's toml configuration as:

```yaml

> MessageQueue:

Reviewer: We renamed this to `MessageBus` in V3
```suggestion
MessageBus:
```

  Protocol: redis
  Host: localhost
  Port: 6379
  Type: redis
```

#### Additional Configuration
Individual client abstractions allow additional configuration properties which can be provided via configuration file:

```yaml

> MessageQueue:

Reviewer: ```suggestion
MessageBus:
```

```

**NOTE**  

> For complete details on configuration options see the [MessageBus documentation](https://docs.edgexfoundry.org/3.0/microservices/general/messagebus/)

Reviewer: ```suggestion
For complete details on configuration options see the [MessageBus documentation](https://docs.edgexfoundry.org/latest/microservices/general/messagebus/)
```

var err error
messageBus, err = msgFactory.NewMessageClient(types.MessageBusConfig{
    Broker:   types.HostInfo{

>     Host:     Configuration.MessageQueue.Host,
>     Port:     Configuration.MessageQueue.Port,
>     Protocol: Configuration.MessageQueue.Protocol,

Reviewer: ```suggestion
    Host:     Configuration.MessageBus.Host,
    Port:     Configuration.MessageBus.Port,
    Protocol: Configuration.MessageBus.Protocol,
```

  },

>   Type: Configuration.MessageQueue.Type,})

Reviewer: ```suggestion
  Type: Configuration.MessageBus.Type,})
```

```go
messageBus, err := factory.NewMessageClient(types.MessageBusConfig{
    Broker:   types.HostInfo{

>     Host:     Configuration.MessageQueue.Host,
>     Port:     Configuration.MessageQueue.Port,
>     Protocol: Configuration.MessageQueue.Protocol,
>   },
>   Type: Configuration.MessageQueue.Type,

Reviewer: ```suggestion
    Host:     Configuration.MessageBus.Host,
    Port:     Configuration.MessageBuss.Port,
    Protocol: Configuration.MessageBus.Protocol,
  },
  Type: Configuration.MessageBus.Type,
```
